README.md

```diff
@@ -298,6 +301,60 @@ All read operations remain available, allowing you to query dashboards, run Prom
 - `--server.tls-cert-file`: Path to TLS certificate file for server HTTPS
 - `--server.tls-key-file`: Path to TLS private key file for server HTTPS
 
+### Dynamic Toolset Discovery
+
+For even more efficient context window usage, you can enable **dynamic toolset discovery** mode with the `--dynamic-toolsets` flag. In this mode, tools are not loaded at startup. Instead, clients can discover available toolsets and selectively enable only the ones they need at runtime.
```

Comment:
```suggestion
For more efficient context window usage, you can enable **dynamic toolset discovery** mode with the `--dynamic-toolsets` flag. In this mode, tools are not loaded at startup. Instead, clients can discover available toolsets and selectively enable only the ones they need at runtime.
```

let's not pretend we're efficient in the first place 😅 

I think we could even be more clear here about our high default token usage, and instead say something like "Grafana supports a lot of functionality, so by default the server exposes a lot of tools which can eat into your context window. For more efficient context window usage, ...".